### DISCUSSION TITLE
Download videos from Benshi.fr

### DISCUSSION THREAD

**Discussion by Angegg31** [aclap-dev/video-downloadhelper#566] [Help and Q&A]
### Your question

Hello, I can no longer download videos from the benshi.fr site, it worked for 4 downloads (including https://benshi.fr/content/la-baleine-et-lescargote-66473). This no longer works. Thank you in advance for your help.

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://benshi.fr/content/monsieur-bout-de-bois-3312",
  "error": {
    "details": "SideDownload error: : [dash @ 0000025628578a00] Error when loading first fragment of playlist\r\n[in#0 @ 0000025628578840] Error opening input: Invalid data found when processing input\r\nError opening input file https://d2123fqla8bkbg.cloudfront.net/stream/6/dash/V2_B001106MonsieurBoutdeBois_hd_vm_feature_V3/V2_B001106MonsieurBoutdeBois_hd_vm_feature_V3.mpd.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_1779677434327774",
    "error": "coapp_failure",
    "id": "coapp_failure_29635662-149b-4c94-9024-08fb8d1963d0"
  }
}

**Comment by Mattmjs** [comment]
It fails on it's own even in ffmpeg, I tried another tool which works but download speed is heavily throttled that I closed the
command prompt. I was left with a partial file and playable. Not DRM protected but the webpage I visited said it needed DRM enabled.
So I don't know what the issue is with the extension or ffmpeg. The tool that worked is **yt-dlp**.
